README.md

```diff
@@ -30,189 +30,7 @@ Our workflow is described in [CONTRIBUTING.rst](CONTRIBUTING.rst).
 
 ## Developer Environment
 
-Please prepare you local development environment as follows.
```

Comment:
This is fine here, but would be cleaner if this change was done in a separate PR since it isn't related to the main topic.

Reply:
Agreed, sorry that was a it sloppy and makes this harder to Backport/patch where needed indeed.

Reply:
Moved this to #453